packages/indexdb/tests/setup.ts

```diff
@@ -16,4 +16,5 @@ Object.defineProperty(globalThis, "crypto", {
   },
 });
 
-Object.assign(global, { TextDecoder, TextEncoder });
+const _TextUtils = JSON.parse(JSON.stringify({ TextDecoder, TextEncoder }))
+Object.assign(global, _TextUtils)
```

Comment:
@chereseeriepa if you stringify `{ TextEncode }` you get `{}`

So perhaps the correct edit is:
```suggestion
```